Update the PPA reference guide to be Diataxis-compliant (#571)
canonical/launchpad-manual: docs/user/reference/packaging/ppas/ppa.rst

.. meta::
   :description: Personal Package Archive reference - supported series
      and architectures, quotas, retention, and signing key facts.

.. _personal-package-archive:

Personal Package Archive
========================

A Personal Package Archive (PPA) is a per-user or per-team apt

> repository hosted by Launchpad, built from uploaded source packages.

Reviewer: ```suggestion
repository hosted by Launchpad, that contains uploaded source packages.
```

Author: The suggest applies an incorrect meaning, even more incorrect than the existing wording was. So I just trimmed the part after the comma and replaced the comma with a full stop.
